lib/ets.ex

```diff
@@ -28,6 +28,8 @@ defmodule Ets do
   @type table_identifier :: table_name | table_reference
   @type match_pattern :: :ets.match_pattern()
   @type match_spec :: :ets.match_spec()
+  @type continuation :: :ets.continuation()
```

Comment:
It looks like `:ets.continuation()` isn't exported:
https://github.com/erlang/otp/blob/master/lib/stdlib/src/ets.erl#L46
but we could match it:
https://github.com/erlang/otp/blob/master/lib/stdlib/src/ets.erl#L53-L55
with something like:
```suggestion
  @type continuation :: end_of_table |
       {:ets.table(), integer(), integer(), :ets.comp_match_spec(), list(), integer()} |
       {:ets.table, any(), any(), integer(), :ets.comp_match_spec(), list(), integer(),  integer()}
```